### PR TITLE
Send button updates  #1313

### PR DESCRIPTION
**Pull request by golangbu**
@jcbrand @licaon-kter  can you please review my pull request.#1313

### CODE REVIEW

File: sass/_chatbox.scss [hunk 1]
```diff
@@ -278,14 +274,19 @@
...
-                position: static;
-                left: var(--send-button-margin);
-                width: 100%;
+
                 background-color: var(--chat-head-color);
-                color: var(--inverse-link-color);
-                font-size: 80%;
-                height: var(--send-button-height);
-                bottom: calc(-var(--send-button-height) - var(--send-button-margin));
+                color: #fff;
```
Comment (jcbrand) [inline sass/_chatbox.scss]: CSS variables should not be replaced with hardcoded values.

File: sass/_chatbox.scss [hunk 2]
```diff
@@ -259,16 +259,12 @@
...
-                width: 100%;
-                border: none;
-                min-height: var(--chat-textarea-height);
-                margin-bottom: -4px; // Not clear why this is necessar :(
-                resize: none;
+		     color: var(--chat-textarea-color);
+		     border: 1px solid #fff;
+                     margin: 0 0 0 0px;
+                     min-height: 40px;
+                     padding: 10px 75px 10px 10px;
```
Comment (jcbrand) [inline sass/_chatbox.scss]: Ideally responsive values (e.g. `0.5em`) should not be replaced with fixed values.

Why was this change necessary?

File: sass/_chatbox.scss [hunk 3]
```diff
@@ -259,16 +259,12 @@
...
-                padding: 0.5em;
-                width: 100%;
-                border: none;
-                min-height: var(--chat-textarea-height);
-                margin-bottom: -4px; // Not clear why this is necessar :(
-                resize: none;
+		     color: var(--chat-textarea-color);
+		     border: 1px solid #fff;
+                     margin: 0 0 0 0px;
+                     min-height: 40px;
```
Comment (jcbrand) [inline sass/_chatbox.scss]: CSS variables should not be replaced with hardcoded values.

**Comment by golangbu** [comment]
@jcbrand  can you review the changes now.I have tried for both show_send_button: true and show_send_button: false. Both are working.

**Comment by poVoq** [comment]
Any change this could be still merged? Really useful on mobile view. Thanks!

**Comment by poVoq** [comment]
Since Fontawesome Icons are loaded by ConverseJS anyways, how about replacing the "send" text with a typical paper-plane icon: https://fontawesome.com/icons/paper-plane?style=solid

**Comment by jcbrand** [comment]
Relevant work done in this commit: 916e300a792ccb6376f72d0560d195c4deadba27
